setup.py

```diff
@@ -19,7 +19,7 @@
 
 def main():
     cpu_count = os.cpu_count() or 8
-    version = "20201111"
+    version = "20201123.ob_msg_refactor"
```

Comment:
I am not sure if it's a good idea to name version this way. Do we need the extra message in it?

Reply:
Right. It was meant for building image for parrot components only. I've remove the extra message.